notebook/static/notebook/js/notebook.js

```diff
@@ -628,6 +628,73 @@ define(function (require) {
         });
         return result;
     };
+    
+    /**
+     * Toggles the marks on the cells
+     * @param  {Cell[]} [cells] - optionally specify what cells should be toggled
+     */
+    Notebook.prototype.toggle_marks = function(cells) {
```

Comment:
call this `toggle_cell_marked` (the word marks has more of a visualization connotation).